Funcky.Test/OptionTest.cs

```diff
@@ -464,6 +464,21 @@ public void InspectDoesNotExecuteSideEffectWhenOptionIsNone()
             Assert.Equal(option, inspectedOption);
         }
 
+        [Fact]
+        public void GivenAFunctionWithADefaultOptionParameterGivesANoneWhenNoValueGiven()
+        {
+            MethodWithDefaultOptionParameter()
```

Comment:
Use `FunctionalAssert.IsNone()`

Reply:
fixed in 9d88d4fb5c71997a25a21b6c8c563f5dcd354b70